spec/find-view-spec.coffee

```diff
@@ -829,6 +829,20 @@ describe 'FindView', ->
       beforeEach ->
         findView.findEditor.focus()
 
+      it "scrolls to the first match depending on the settings", ->
```

Comment:
Can you make this two specs? One with the setting on, and one with it off.

Also, this test doesnt look like it is testing that the scrollTop changes / doesnt change.